{"metadata":{"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"io","kind":"identifier"}],"symbolKind":"case","externalID":"s:13SwiftErrorKit06SourceB9AttributeO2ioyA2CmF","role":"symbol","title":"SourceErrorAttribute.io","modules":[{"name":"SwiftErrorKit"}],"roleHeading":"Case"},"schemaVersion":{"major":0,"minor":3,"patch":0},"hierarchy":{"paths":[["doc:\/\/SwiftErrorKit\/documentation\/SwiftErrorKit","doc:\/\/SwiftErrorKit\/documentation\/SwiftErrorKit\/SourceErrorAttribute"]]},"primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["macOS"],"tokens":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"io","kind":"identifier"}],"languages":["swift"]}]}],"kind":"symbol","sections":[],"identifier":{"url":"doc:\/\/SwiftErrorKit\/documentation\/SwiftErrorKit\/SourceErrorAttribute\/io","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"Communication error."},{"text":" ","type":"text"},{"type":"text","text":"Network outage, connectivity problem, protocol error, …"}],"variants":[{"paths":["\/documentation\/swifterrorkit\/sourceerrorattribute\/io"],"traits":[{"interfaceLanguage":"swift"}]}],"references":{"doc://SwiftErrorKit/documentation/SwiftErrorKit":{"type":"topic","role":"collection","abstract":[{"type":"text","text":"Error attribution and commons."}],"url":"\/documentation\/swifterrorkit","identifier":"doc:\/\/SwiftErrorKit\/documentation\/SwiftErrorKit","kind":"symbol","title":"SwiftErrorKit"},"doc://SwiftErrorKit/documentation/SwiftErrorKit/SourceErrorAttribute/io":{"type":"topic","role":"symbol","abstract":[{"type":"text","text":"Communication error."},{"text":" ","type":"text"},{"type":"text","text":"Network outage, connectivity problem, protocol error, …"}],"url":"\/documentation\/swifterrorkit\/sourceerrorattribute\/io","identifier":"doc:\/\/SwiftErrorKit\/documentation\/SwiftErrorKit\/SourceErrorAttribute\/io","kind":"symbol","fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"io"}],"title":"SourceErrorAttribute.io"},"doc://SwiftErrorKit/documentation/SwiftErrorKit/SourceErrorAttribute":{"url":"\/documentation\/swifterrorkit\/sourceerrorattribute","role":"symbol","navigatorTitle":[{"text":"SourceErrorAttribute","kind":"identifier"}],"type":"topic","abstract":[{"type":"text","text":"An error attribute to include information about the high-level component"},{"text":" ","type":"text"},{"text":"in which the error emerged.","type":"text"}],"fragments":[{"kind":"keyword","text":"enum"},{"text":" ","kind":"text"},{"text":"SourceErrorAttribute","kind":"identifier"}],"kind":"symbol","identifier":"doc:\/\/SwiftErrorKit\/documentation\/SwiftErrorKit\/SourceErrorAttribute","title":"SourceErrorAttribute"}}}